### PR TITLE
Update chromebook.mdx

### PR DESCRIPTION
**Pull request by Owen-sz**
Change the manual installation for raw images to an easier method

### CODE REVIEW

File: src/content/docs/en/anywhere/chromebook.mdx [hunk 1]
```diff
@@ -98,25 +98,24 @@ Chromebook Edition will be released soon, installation guide for that will be fo
   pet, etc.
 </Alert>
 
-This guide assumes you are on x86 and have [Developer Mode](https://chromium.googlesource.com/chromiumos/docs/+/main/developer_mode.md#dev-mode) and [USB Booting Enabled](https://chromium.googlesource.com/chromiumos/docs/+/main/developer_guide.md#boot-from-your-usb-disk). There are alterations to these instructions you could make and still get the same end result, this is just what we find to be the easiest way.
+This guide assumes you are on x86 and have [Developer Mode](https://chromium.googlesource.com/chromiumos/docs/+/main/developer_mode.md#dev-mode) and [USB Booting Enabled](https://chromium.googlesource.com/chromiumos/docs/+/main/developer_guide.md#boot-from-your-usb-disk). We also assume you are using gparted for the partitioning tasks, but other tools will work too. There are alterations to these instructions you could make and still get the same end result, this is just what we find to be the easiest way.
```
Comment (nothingneko) [inline src/content/docs/en/anywhere/chromebook.mdx:101]:
```suggestion
This guide assumes you are on x86 and have [Developer Mode](https://chromium.googlesource.com/chromiumos/docs/+/main/developer_mode.md#dev-mode) and [USB Booting Enabled](https://chromium.googlesource.com/chromiumos/docs/+/main/developer_guide.md#boot-from-your-usb-disk). We also assume you are using GParted for partitioning tasks. There are alterations you could make to these instructions and still get the same result, this is just what we think is the easiest way.
```

File: src/content/docs/en/anywhere/chromebook.mdx [hunk 2]
```diff
@@ -98,25 +98,24 @@ Chromebook Edition will be released soon, installation guide for that will be fo
...
 
 ### Raw (.img) images:
 
 1. Download, decompress, and flash an [Ultramarine Raw Image](https://images.fyralabs.com/images/ultramarine/) with a desktop environment (for example, flagship-base-disk-x86_64.img), to a USB Drive or SD Card smaller then the internal disk of your Chromebook.
-2. Delete the first partition on the disk.
-3. Create a 16MB ext4 partition.
-4. Flash the [submarine.kpart](https://nightly.link/FyraLabs/submarine/workflows/build/main/submarine-x86_64.zip) file to the partition.
-5. Set the partition flags with cgpt (may have to install `vboot-utils`).
+2. Create a 16 MB ext4 partition at the END of the disk (for proper resizing after installation). In gparted, accomplish this by making a new partition in the unallocated space, and dragging it all the way to the right while still in the 'Create new Partition' menu. If gparted gives you a 'Libparted Warning,' select 'Fix.'
```
Comment (nothingneko) [inline src/content/docs/en/anywhere/chromebook.mdx:106]:
```suggestion
2. Create a 16 MB ext4 partition at the END of the disk (for proper resizing after installation). In GParted, do this by making a new partition in the unallocated space, and dragging it all the way to the right while still in the 'Create new Partition' menu. If GParted gives you a 'Libparted Warning,' select 'Fix.'
```

File: src/content/docs/en/anywhere/chromebook.mdx [hunk 3]
```diff
@@ -98,25 +98,24 @@ Chromebook Edition will be released soon, installation guide for that will be fo
...
 ```
-cgpt add -i <partition number> -t kernel -P 15 -T 1 -S 1 /dev/sdX
+cgpt add -i <Submarine partition number> -t kernel -P 15 -T 1 -S 1 /dev/sdX
 ```
 
-6. Boot the image on the Chromebook.
+5. Boot the image on the Chromebook.
 
-You're now inside the Ultramarine install on your USB/SD card.
+You're now inside the Ultramarine install on your USB/SD card. Running slowly is standard behavior and is not reflective of the preformance once running off internal storage.
```
Comment (nothingneko) [inline src/content/docs/en/anywhere/chromebook.mdx:116]:
```suggestion
You're now inside the Ultramarine install on your USB/SD card. Slowness and lag is expected behavior and does not reflect how Ultramarine will run from internal storage.
```

File: src/content/docs/en/anywhere/chromebook.mdx [hunk 4]
```diff
@@ -98,25 +98,24 @@ Chromebook Edition will be released soon, installation guide for that will be fo
...
 
-7. `dd` the USB/SD card to the internal disk of the Chromebook. (Note, you do NOT need to resize the root filesystem, it should do this automatically upon first boot).
+6. `dd` the USB/SD card to the internal disk of the Chromebook. (Note, you do NOT need to resize the root filesystem, it should do this automatically upon first boot).
```
Comment (nothingneko) [inline src/content/docs/en/anywhere/chromebook.mdx:118]:
```suggestion
6. Use `dd` to copy the contents of the USB/SD card to the internal storage of the Chromebook. (Note, you do NOT need to resize the root filesystem, it should do this automatically upon first boot).
```
also put this note in an info alert i'm too lazy to do it on my phone

File: src/content/docs/en/anywhere/chromebook.mdx [hunk 5]
```diff
@@ -130,7 +129,7 @@ dd if=/dev/sda of=/dev/mmcblk0 bs=4M status=progress
 
 If you're using other devices, adjust accordingly. If your're using an SD card to install to eMMC, or a USB stick to install to UFS, both devices will show as 'mmcblkX' (mmcblk 0/1/2), and 'sdX' (sd a/b/c). Adjust the numbers/letters accordingly.
 
-8. Once the `dd` is done, verify the Submarine partition shows 'ChromeOS Kernel' as a flag on the internal disk. We recommend using GParted for this (dnf install gparted).
+7. Once the `dd` is done, verify the Submarine partition shows 'ChromeOS Kernel' as a flag on the internal disk. We recommend using GParted for this (dnf install gparted if needed).
```
Comment (nothingneko) [inline src/content/docs/en/anywhere/chromebook.mdx:132]:
```suggestion
7. Once `dd` is done, verify the Submarine partition on the internal storage shows 'ChromeOS Kernel' as a flag. You can use the Disks app on Flagship, GNOME, and Xfce, or, KDE Partition Manager on KDE. Simply select the disk and partition, the label should appear under the logical view.
```

**Review by nothingneko — CHANGES_REQUESTED** [review]

File: src/content/docs/en/anywhere/chromebook.mdx [hunk 6]
```diff
@@ -98,25 +98,24 @@ Chromebook Edition will be released soon, installation guide for that will be fo
...
 
 ### Raw (.img) images:
 
 1. Download, decompress, and flash an [Ultramarine Raw Image](https://images.fyralabs.com/images/ultramarine/) with a desktop environment (for example, flagship-base-disk-x86_64.img), to a USB Drive or SD Card smaller then the internal disk of your Chromebook.
-2. Delete the first partition on the disk.
-3. Create a 16MB ext4 partition.
-4. Flash the [submarine.kpart](https://nightly.link/FyraLabs/submarine/workflows/build/main/submarine-x86_64.zip) file to the partition.
-5. Set the partition flags with cgpt (may have to install `vboot-utils`).
+2. Create a 16 MB ext4 partition at the END of the disk (for proper resizing after installation). In GParted, do this by making a new partition in the unallocated space, and dragging it all the way to the right while still in the 'Create new Partition' menu. If GParted gives you a 'Libparted Warning,' select 'Fix.'
```
Comment (nothingneko) [inline src/content/docs/en/anywhere/chromebook.mdx:106]:
```suggestion
2. Create a 16MB partition at the END of the disk (for proper resizing after installation). In GParted, do this by making a new partition in the unallocated space, and dragging it all the way to the right while still in the 'Create new Partition' menu. If GParted gives you a 'Libparted Warning,' select 'Fix.'
```
doesn't need to be ext4, subm will change it anyway

File: src/content/docs/en/anywhere/chromebook.mdx [hunk 7]
```diff
@@ -98,25 +98,24 @@ Chromebook Edition will be released soon, installation guide for that will be fo
...
 ```
 
-6. Boot the image on the Chromebook.
+5. Boot the image on the Chromebook.
 
-You're now inside the Ultramarine install on your USB/SD card.
+You're now inside the Ultramarine install on your USB/SD card. Slowness and lag is expected behavior and does not reflect how Ultramarine will run from internal storage.
 
-7. `dd` the USB/SD card to the internal disk of the Chromebook. (Note, you do NOT need to resize the root filesystem, it should do this automatically upon first boot).
+6. Use `dd` to copy the contents of the USB/SD card to the internal storage of the Chromebook. (Note, you do NOT need to resize the root filesystem, it should do this automatically upon first boot).
```
Comment (nothingneko) [inline src/content/docs/en/anywhere/chromebook.mdx:118]: put this note in an info alert

File: src/content/docs/en/anywhere/chromebook.mdx [hunk 8]
```diff
@@ -139,8 +138,8 @@ If you're using other devices, adjust accordingly. If your're using an SD card t
 3. Plug in both drives you just flashed to the Chromebook (a USB hub works!).
 4. Boot to the installer's live emvironment by pressing <kbd>Ctrl</kbd>+<kbd>U</kbd> (You shouldn't boot to a boot option that lists a kernel version (ex ultramarine-linux-6.10.0-X-X-X), just boot to 'Ultramarine Linux _EDITION_.' If you still have ChromeOS on the internel disk, it is likely that this will be the 14th entry, if not, it may be the 3rd). If it does not boot, restart and try a different Ultramarine Linux entry.
 5. In the live enviornment of the ISO, install as normal.
-6. Once the install is done, return to the live environment desktop. Delete `/boot/efi` (the one with boot flags), make a new 16MiB EXT4 partition.
-7. flash [submarine.kpart](https://nightly.link/FyraLabs/submarine/workflows/build/main/submarine-x86_64.zip) to the new EXT4 partition, and set flags with cgpt (may have to install `vboot-utils`).
+6. Once the install is done, return to the live environment desktop. Make a new 16MiB ext4 partition, this can be done by resizing the 'root' partition (the largest one).
```
Comment (nothingneko) [inline src/content/docs/en/anywhere/chromebook.mdx:141]: suggest creating that partition in anaconda?

File: src/content/docs/en/anywhere/chromebook.mdx [hunk 9]
```diff
@@ -139,8 +138,8 @@ If you're using other devices, adjust accordingly. If your're using an SD card t
...
+7. flash [submarine.kpart](https://nightly.link/FyraLabs/submarine/workflows/build/main/submarine-x86_64.zip) to the new ext4 partition, and set flags with cgpt (may have to install `vboot-utils`).
```
Comment (nothingneko) [inline src/content/docs/en/anywhere/chromebook.mdx:142]: doesn't need to be ext4
```suggestion
7. flash [submarine.kpart](https://nightly.link/FyraLabs/submarine/workflows/build/main/submarine-x86_64.zip) to the new partition, and set flags with cgpt (may have to install `vboot-utils`).
```

**Review by nothingneko — APPROVED** [review]

**Comment by nothingneko** [comment]
did not mean to approve, am on mobile, take this comment as requesting the changes

File: src/content/docs/en/anywhere/chromebook.mdx (hunk 8 above)
Comment (Owen-sz) [reply src/content/docs/en/anywhere/chromebook.mdx:141]: Would work, but this is probably an easier method on the user side. Might play around with it and see if manual partitioning can be made easier.

**Review by nothingneko — APPROVED** [review]

**Pull request merged**